packages/bot-web-ui/src/app/app.jsx

```diff
@@ -47,12 +48,24 @@ const App = ({ passthrough }) => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [onMount, onUnmount]);
 
+    React.useEffect(() => {
+        window.addEventListener('offline', onDisconnectFromNetwork);
+        return () => {
+            window.removeEventListener('offline', onDisconnectFromNetwork);
+        }
+    }, []);
+
+    const onDisconnectFromNetwork = () => {
+        setIsLoading(false);
+    }
```

Comment:
Please move this function inside useEffect to avoid eslint warning

Reply:
done 👍